models/sipnet/R/met2model.SIPNET.R

```diff
@@ -147,7 +147,9 @@ met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date
       }
       
       Rain <- ncdf4::ncvar_get(nc, "precipitation_flux")
-      pres <- ncdf4::ncvar_get(nc,'air_pressure') ## in pascal
+      
+      press <- ncdf4::ncvar_get(nc,'air_pressure') ## in pascal
+
```

Comment:
Whether this change is an improvement is an aesthetic question, but I recommend reverting this change -- the PR is big enough already and including whitespace changes in otherwise-untouched files just makes it bigger.